tgui/packages/tgui/interfaces/ComputerSimple.js

```diff
@@ -0,0 +1,36 @@
+import { Fragment } from 'inferno';
+import { useBackend } from "../backend";
+import { Button, LabeledList, Section, Box, Table, ProgressBar } from "../components";
+import { LabeledListItem } from "../components/LabeledList";
+import { Window } from "../layouts";
+
+export const ComputerSimple = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window theme="syndicate">
+      <Window.Content>
+        {data.rows.map(record => (
+          <Section key={record.title} title={record.title}
+            buttons={
```

Comment:
I feel like there should be a var in `record` itself to signify whether buttons should be present or not, incase theres just a section header.

Reply:
Originally I thought about doing that, but decided not to because I could just use whether buttontitle exists or not to also decide if there should be a button. A buttonexists var would be redundant.
What I'd prefer to do is not have the buttons={} section even have a button defined if buttontitle is null, but I haven't worked out a way to actually do that without having to duplicate the entire <Section>, which seemed like a very bad idea.